### PR TITLE
DRF-Blog-App:Blog app with summernote editor.

### PR DESCRIPTION
**Pull request by VISHU1999**
# Simple Blog App with summer editor 

Test: https://test.pypi.org/project/drf-blog/1.1/

### CODE REVIEW

File: blog/serializers.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+from blog.models import Blog
+
+
+class BlogSerializer(serializers.ModelSerializer):
+    author = serializers.SerializerMethodField()
+    read_time = serializers.ReadOnlyField()
+
+    class Meta:
+        model = Blog
+        fields = "__all__"
```
Comment (Pradhu-MAN) [inline blog/serializers.py]: change this all to name of the fields

File: blog/models.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
+from django.contrib.auth import get_user_model
+from django.db import models
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
```
Comment (Pradhu-MAN) [inline blog/models.py]: we are using this but not creating translation files.

File: blog/models.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    file_path = "blog/{author_id}/{title}-{filename}".format(
+        author_id=str(instance.author.id), title=str(instance.title), filename=filename
+    )
+    return file_path
+
+
+class BlogState(models.TextChoices):
+    draft = (
+        "draft",
+        _("Draft"),
```
Comment (Pradhu-MAN) [inline blog/models.py]: _ is not needed on our admin side.

File: blog/models.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    )
+
+    published = (
+        "published",
+        _("Published"),
```
Comment (Pradhu-MAN) [inline blog/models.py]: same here

File: blog/models.py [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        abstract = True
+
+    def save(self, *args, **kwargs):
+        self.updated_at = timezone.now()
+        return super().save(*args, **kwargs)
+
+
+class Blog(CreatedUpdatedMixin):
+    title = models.CharField(max_length=60)
+    content = models.TextField(_("Add Blog Content"))
```
Comment (Pradhu-MAN) [inline blog/models.py]: same here

Comment (Pradhu-MAN) [reply blog/models.py]: or if we are adding it we should have the translations as well.

File: blog/tests.py [hunk 6]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
```
Comment (Pradhu-MAN) [inline blog/tests.py]: may be adding test cases would give us a lot of confidence on this library

**Review by Pradhu-MAN — CHANGES_REQUESTED** [review]
Few comments added, Good Work!

File: blog/tests.py (hunk 6 above)
Comment (VISHU1999) [reply blog/tests.py]: @Pradhu-MAN Thanks for your feedback 
I will Update the PR with your required changes

**Pull request merged**